### ISSUE TITLE
An Its() function cannot access index of zero.

### ISSUE THREAD

**Issue by torn2537** [cypress-io/cypress#6216]
### Current behavior:

From the tutorial on the official website (https://docs.cypress.io/api/commands/its.html) when I have used with an array following the example I have found an issue of the function. The function could access indices from indices `one` to last indices of the array. However, when I have used the indices `zero` with the function the compiler have an error.

### Desired behavior:

The function have been access by  indices `zero` to the last indices of the example array.

### Test code to reproduce
```js
cy.wrap(['Wai Yan', 'Yu']).its(1).should('eq', 'Yu') // true
cy.wrap(['Wai Yan', 'Yu']).its(0).should('eq', 'Yu') // CypressError: cy.its() expects the propertyName argument to have a value.
```

### Versions

Cypress version (Installed via npm) : 3.8.2
Os: Windows 10

**Comment by datner** [comment]
I am currently doing this one, the project is a bit overwhelming for me so I might get this wrong so please be patient with me :) (also writing coffeescript on the fly so thats cool).

I'm actually done but the pre-commit hook is breaking on me, so, I'm dealing with that

**Comment by flotwig** [comment]
Released in [`4.0.0`](https://on.cypress.io/changelog#4-0-0).
